.github/workflows/build-test.yml

```diff
@@ -18,4 +22,18 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Test
-        run: go test ./...
\ No newline at end of file
+        run: go test ./...
+
+      - name: Build
+        run: go run .
+        working-directory: examples/
```

Comment:
@RamanaReddy0M  , this step can be renamed as `Run Example` and we should add a new step `Run` which will run code at `/cmd/cdncheck/` 

- also suggesting adding a race condition step 
```
go run -race . -i scanme.sh,projectdiscovery.io
```